src/os/bluestore/ceph_aio.h

```diff
@@ -22,13 +22,13 @@ struct aio_t {
 #if defined(HAVE_LIBAIO)
   struct iocb iocb{};  // must be first element; see shenanigans in aio_queue_t
 #elif defined(HAVE_POSIXAIO)
-  //  static long aio_listio_max = -1;
   union {
     struct aiocb aiocb;
     struct aiocb *aiocbp;
   } aio;
   int n_aiocb;
 #endif
+  CephContext *cct;
```

Comment:
i am not sure if we want to add yet another 8-byte member variable in `aio_t`. probably we can enable it only in debug build?

@liewegas what do you think?

Reply:
@tchaikov 
I want the context there so I can do logging from the aio-functions.
Doing so in only debug builds is workable for the time being. 
Not sure what to do when we need to diagnose non-debug external runs.

Reply:
@tchaikov 
I'll remove the logging.
Would it be valid to write the errors to stderr?? That way there is some indication that things are not oke...

Reply:
@wjwithagen  if the daemon is daemonized, stderr will be closed. i am afraid we cannot rely on `stderr` for logging.